tests/api/printf.cpp

```diff
@@ -153,29 +180,16 @@ TEST_F(WithCommandQueue, TooLongPrintf) {
     size_t lws = 1;
     EnqueueNDRangeKernel(kernel, 1, nullptr, &gws, &lws, 0, nullptr, nullptr);
     Finish();
-
-    releaseStdout(fd);
-    auto printf_buffer = getStdoutContent();
-    ASSERT_NE(printf_buffer, nullptr);
-
-    // We only get the first 2 prints because the buffer is too small to get the
-    // last one.
-    const char* message = "get_global_id(0) = 0\nget_global_id(1) = 0\n";
-    ASSERT_STREQ(printf_buffer, message);
+    // Reset the buffer if complete, otherwise keep the remaining part
+    ASSERT_STREQ(globalBuffer.data(), message);
 }
 
-TEST_F(WithCommandQueue, TooLongPrintf2) {
-    // each print takes 12 bytes (4 for the printf_id, and 2*4 for the 2 integer
-    // to print) + 4 for the byte written counter + 8 which are not enough for
-    // the third print, but should not cause any issue in clvk
-    auto cfg1 =
-        CLVK_CONFIG_SCOPED_OVERRIDE(printf_buffer_size, uint32_t, 36, true);
-
-    temp_folder_deletion temp;
-    stdoutFileName = getStdoutFileName(temp);
-
-    int fd;
-    ASSERT_TRUE(getStdout(fd));
+TEST_F(WithPrintfEnabled, TooLongPrintf2) {
+    auto props = setup_arm_printf_test(46);
```

Comment:
Why `46`? it used to be `36` with a comment explaining why, but now it's another value that we don't know where it comes from.